models/game.js

```diff
@@ -30,87 +30,30 @@ $(document).ready(function () {
         if (dragonNum === playerNum) {
             // Subtract the damage amount from the dragon's health.
             dragonHealth -= playerDamage;
-            $("#combatText").text("You slash the dragon with " + playerDamage + " damage");
+            $("#combatText").text("Clean hit! You slashed the dragon for " + playerDamage + " damage");
         }
         else {
             // Subtract the damage amount from the player's health.
             playerHealth -= dragonDamage;
-            $("#combatText").text("OH NO! The dragon slashed you with " + dragonDamage + " damage");
+            $("#combatText").text("Damn! The dragon slashed you for " + dragonDamage + " damage");
 
         }
+
+        // update characters health after each round
         $("#playerHealthBar").text("Your health: " + playerHealth);
         $("#bossHealthBar").text("Boss health: " + dragonHealth);
+
+        if (playerHealth <= 0) {
+            // the boss wins
+            $("#combatText").text("You've been slain... Your story ends here...");
+            // prevent button from working after the game ends
+            $("#attackBtn").attr("onclick", "").unbind("click");
+        }
+        if (dragonHealth <= 0) {
+            // the player wins
+            $("#combatText").text("Victory! You've slain the dragon!");
+            // prevent button from working after the game ends
+            $("#attackBtn").attr("onclick", "").unbind("click");
+        }
     });
 });
-
-    // function checkRound() {
-
-    //     // If the player has less than 0 health.... then the player lost.
-    //     if (playerHealth <= 0) {
-    //         $("#combatText").text("You've been slain... Your story ends here...");
-    //        
-    //         // Exit the game
-    //         process.exit();
-    //     }
-
-    //     // If the dragon has less than 0 health.... then the player won.
-    //     if (dragonHealth <= 0) {
-    //         $("#combatText").text("Victory! You've slain the dragon!");
-    //          
-    //         // Exit the game
-    //         process.exit();
-    //     }
-
-    //     // After performing the "check", the next round is initiated.
-    //     playRound();
-
-    // }
-
-    // // =======================================================================================================
-    // // This function holds the game logic
```

Comment:
why is game logic commented out?

Reply:
That was the code from ZombieChallenge that I was trying to edit to fit into our project. It wasn't working as intended so I threw all that code out and started from scratch. The page is working now without all the excess code.